R/conversion.R

```diff
@@ -294,3 +316,18 @@ py_to_r.pandas.core.frame.DataFrame <- function(x) {
   df
   
 }
+
+#' @export
+summary.pandas.core.frame.DataFrame <- function(object, ...) {
+  summary.pandas.core.series.Series(object, ...)
```

Comment:
FWIW this doesn't seem like a very natural delegation (although the code is correct). Why not just have this and the other `summary` method call a helper `py_describe()` function?

Reply:
The other thing we do elsewhere in the package is just assign the method we will delegate to directly, e.g. https://github.com/rstudio/reticulate/blob/master/R/python.R#L285-L286

Reply:
I like JJ's idea. @kevinushey Do you mind if I adopt this and delegate directly?

Reply:
Sounds good!